### PR TITLE
Add audio classification for speechbrain

### PR DESCRIPTION
**Pull request by osanseviero**
This PR shows how to do audio classification with Speechbrain. This enables inference for existing repos such as
* Language identification: https://huggingface.co/speechbrain/lang-id-commonlanguage_ecapa
* Command recognition: https://huggingface.co/speechbrain/google_speech_command_xvector
* Speaker recognition: https://huggingface.co/speechbrain/spkrec-xvect-voxceleb

FYI @patrickvonplaten @anton-l @TParcollet

### CODE REVIEW

File: api-inference-community/docker_images/speechbrain/app/pipelines/audio_classification.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import numpy as np
+import torch
+from app.pipelines import Pipeline
+from speechbrain.pretrained import EncoderClassifier
+
+
+class AudioClassificationPipeline(Pipeline):
+    def __init__(self, model_id: str):
+        self.model = EncoderClassifier.from_hparams(source=model_id)
```
Comment (TParcollet) [inline api-inference-community/docker_images/speechbrain/app/pipelines/audio_classification.py:11]: Here, as discussed, I wonder if it wouldn't be better to load a class given as an argument to the config.json.

File: api-inference-community/docker_images/common/tests/test_api.py [hunk 2]
```diff
@@ -17,6 +17,7 @@
 
 ALL_TASKS = {
     "audio-to-audio",
+    "audio-classification",
```
Comment (mishig25) [inline api-inference-community/docker_images/common/tests/test_api.py:20]: Nit: if we are trying to preserve alphabetical order here
```suggestion
    "audio-classification",
    "audio-to-audio",
```

File: api-inference-community/docker_images/common/tests/test_api_audio_classification.py [hunk 3]
```diff
@@ -39,7 +39,7 @@ def read(self, filename: str) -> bytes:
             bpayload = f.read()
         return bpayload
 
-    def test_simple(self):
+    def test_original_audiofile(self):
```
Comment (mishig25) [inline api-inference-community/docker_images/common/tests/test_api_audio_classification.py:42]: Out of curiosity, what is the difference between naming conventions of: `test_simple` vs `test_original_audiofile`?

File: api-inference-community/docker_images/speechbrain/app/pipelines/audio_classification.py (hunk 1 above)
Comment (osanseviero) [reply api-inference-community/docker_images/speechbrain/app/pipelines/audio_classification.py:11]: Yes, there will be a larger refactoring P of all `speechbrain` to load the class specified the `config.json`.  We want to make an announcement of `audio-classification` today so we're trying to fastrack this PR to have working Inference API and I'll follow up this week with the change to the specified class.

File: api-inference-community/docker_images/common/tests/test_api_audio_classification.py (hunk 3 above)
Comment (osanseviero) [reply api-inference-community/docker_images/common/tests/test_api_audio_classification.py:42]: `test_original_audiofile` is what's used in ASR, so I went with that one for consistency. (I now realize `audio-to-audio` is inconsistent with ASR. `audio-source-separation` is deprecated but due to the diff GitHub is marking as renaming

Comment (osanseviero) [reply api-inference-community/docker_images/common/tests/test_api_audio_classification.py]: I renamed these for consistency

**Review by Narsil — APPROVED** [review]
LGTM, maybe small nit about the test.

**Review by patrickvonplaten — APPROVED** [review]

**Review by mishig25 — APPROVED** [review]
🚀

**Comment by mishig25** [comment]
@osanseviero lets merge it unless you wanna quickly take care of https://github.com/huggingface/huggingface_hub/pull/312#discussion_r702770252

**Pull request merged**